src/components/menuBar/menu-bar.spec.js

```diff
@@ -417,27 +419,24 @@ describe('material.components.menuBar', function() {
         expect(menuItem.children[0].style.display).toBeFalsy();
         expect(button.getAttribute('aria-checked')).toBe('true');
       }));
+    });
 
-      function setup(attrs) {
-        attrs = attrs || '';
-
-        var template = '<md-menu-item type="radio" ' + attrs + '>Test Item</md-menu-item>';
-
-        var radioMenuItem;
-        inject(function($compile, $rootScope) {
-          // We need to have a `md-menu-bar` as a parent of our menu item, because the menu-item
-          // is only wrapping and indenting the content if it's inside of a menu bar.
-          var menuBarMock = angular.element('<md-menu-bar>');
-          var itemEl = angular.element(template);
+    function setupMenuItem(attrs) {
+      // We need to have a `md-menu-bar` as a parent of our menu item, because the menu-item
+      // is only wrapping and indenting the content if it's inside of a menu bar.
+      var menuBar;
+      var template = '' +
```

Comment:
Interesting. It seems to work for me (Webstorm)

Also we don't do this in other components. But it should be fine, just very nitpicky

- https://github.com/angular/material/blob/master/src/components/autocomplete/autocomplete.spec.js#L590